templates/nextflow-aurora-mysql.yaml

```diff
@@ -103,6 +106,16 @@ Resources:
         PasswordLength: 16
       KmsKeyId: !Sub ${AuroraKeyStack.Outputs.Key}
 
+  AuroraMasterSecretRotationSchedule:
```

Comment:
I don't think you address the KMS permission issue that I described [here](https://github.com/Sage-Bionetworks-Workflows/nextflow-infra/pull/168#pullrequestreview-1370106587).

Reply:
ahh yes, sorry about that.  I've updated and I think it should work now.  here are my updates..

* Update kms-key template to v0.6.83 which contains
an KeyArn[1] as an output and more expansive permission[2].

* Add master secrets to the rotation schedule.  The docs for
the MasterSecretKmsKeyArn[3] says that..
```
CloudFormation grants the execution role for the Lambda
rotation function Decrypt, DescribeKey, and GenerateDataKey
permission to the key in this property
```

[1] https://github.com/Sage-Bionetworks/aws-infra/commit/04c5b11c4907c76917e8a13b3c03779dccd0d1ca
[2] https://github.com/Sage-Bionetworks/aws-infra/commit/177f63228cdb5e6a2933b88b6049ba01a372c0ae
[3] https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-secretsmanager-rotationschedule-hostedrotationlambda.html#cfn-secretsmanager-rotationschedule-hostedrotationlambda-mastersecretarn